Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorHelpers.cpp

```diff
@@ -173,6 +174,13 @@ namespace AzToolsFramework
             }
         }
 
+        // Allow entity system to override the selection
+        auto editorInteractionInterface = AZ::Interface<EditorInteractionInterface>::Get();
+        if (editorInteractionInterface != nullptr)
```

Comment:
uber nit: This could use an if initializer 🙂 (or just move the assignment into the if)

`if (auto editorInteractionInterface = AZ::Interface<EditorInteractionInterface>::Get())`

(lessens scope)